docs/reference/box.md

```diff
@@ -0,0 +1,147 @@
+# Box Connector
+
+The [Elastic Box connector](../connectors/sources/box.py) is built with the Elastic connectors Python framework and is available as a self-managed [connector client](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Box** tile from the connectors list OR **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Box API Authorization
+
+### Box Free Account
+
+#### Create Box User Authentication (OAuth 2.0) Custom App
+
+You'll need to create an OAuth app in the Box developer console by following these steps:
+
+1. Register a new app in the [Box dev console](https://app.box.com/developers/console) with custom App and select User authentication (OAuth 2.0).
+
+2. Add your Elasticsearch endpoint URL to **Redirect URIs**.
```

Comment:
Why Elasticsearch endpoint?

Reply:
because it will redirect to user where he/she needs to configure the connector.

Reply:
Because it will redirect the user to where he or she needs to configure the connector.

Reply:
I'm afraid the Elasticsearch endpoint can't be an HTML page. Elasticsearch exposes only API afaik. 

Also, how can the customer grab an `authorization code` if they are redirected to another page? Usually, a page should handle the `authorization code` and use it.

Reply:
Okay then I have removed this step and now user will redirect to default location

Reply:
Can you please describe how customers can find `authorization code`? 

As far as I understand, the authorization code is a part of the redirect payload so the customers should be able to see it. We need to list those steps. 

I was able to grab the code only when I created a local web server and used localhost as a redirect page. Do you have other options?

Reply:
I have updated the doc to find `authorization code`